src/components/Tiles/_TopServices.scss

```diff
@@ -51,3 +51,91 @@
     margin-bottom: $coa-spacing-large;
   }
 }
+
+
+.coa-TopServicesDepartment {
+  &.floating {
+    padding: 2rem 0;
+    margin-left: 2rem;
+    margin-right: 2rem;
+    margin-top: -70rem;
+    @include media($coa-medium-screen) {
+
+      border-radius: 0;
+    }
+    @include media($coa-large-screen) {
+      border-radius: 6px;
+    }
+  }
+
+  background: $coa-color-primary;
+  padding: ($coa-spacing-large * 1.5) 0;
+  border-radius: 0;
+
+  @include media($coa-medium-screen) {
+    max-width: 100%;
+    margin-top: 0rem;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .coa-TileGroup__tile {
+    margin-bottom: $coa-spacing-small;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @include media($coa-medium-screen) {
+      margin-bottom: initial;
+    }
+  }
+}
+
+
+.coa-TopServicesHome {
+  &.floating {
+    padding: 2rem 0;
+    margin-left: 2rem;
+    margin-right: 2rem;
+    margin-top: -70rem;
+    @include media($coa-medium-screen) {
+
+      border-radius: 0;
+    }
+    @include media($coa-large-screen) {
+      border-radius: 6px;
+    }
+  }
+
+  .coa-TopServices__title {
+    color: $coa-color-decently-black;
+  }
+
+  background: $coa-color-white;
+  padding: ($coa-spacing-xlarge * 1.5) 0;
+  border-radius: 0;
+
+
+  @include media($coa-medium-screen) {
+    max-width: 100%;
+    margin-top: 0rem;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .coa-TileGroup__tile {
+    margin-bottom: $coa-spacing-small;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @include media($coa-medium-screen) {
+      margin-bottom: initial;
+    }
+  }
+}
+
+
+
```

Comment:
These are the new styles for the Top Services for the Home page and the Department page.